packages/cmd/src/CommandProcessor.ts

```diff
@@ -30,10 +30,11 @@ import { Logger } from "../../logger";
 import { IInvokeCommandParms } from "./doc/parms/IInvokeCommandParms";
 import { ICommandProcessorParms } from "./doc/processor/ICommandProcessorParms";
 import { ImperativeExpect } from "../../expect";
-import { inspect, isString } from "util";
```

Comment:
Thanks for replacing `isString`. 
Seems like it was deprecated since node 4

Source: https://nodejs.org/api/util.html#utilisstringobject